tests/private-error-cases.test.ts

```
@@ -59,6 +59,64 @@ afterAll(async () => {
+        device_id: deviceId,
+        platform: 'android',
+        version_name: '1.0.0',
+      }),
+    })
+
+    expect(response.status).toBe(401)
+    const data = await response.json() as { error: string }
+    expect(data.error).toBe('not_authorized')
+
+    const { data: device, error: deviceError } = await getSupabaseClient().from('devices')
+      .select('device_id')
```

ESLint

[error] Should not have line breaks between items, in node CallExpression

(antfu/consistent-chaining)